### PR TITLE
Added missing translations

### PR DESCRIPTION
**Pull request by kristelvdakker**
- Added missing translations to the dutch translation file
- Converted some double quotes to single quotes to keep code styling consistent
- Put some translations in alphabetical order to keep them in line with the ra-translation-file in React Admin

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -8,37 +8,38 @@ const dutchMessages: TranslationMessages = {
             back: 'Ga terug',
             bulk_actions: '1 geselecteerd |||| %{smart_count} geselecteerd',
             cancel: 'Annuleren',
+            clear_array_input: 'lijst wissen',
```
Comment (nickwaelkens) [inline src/index.ts:11]:
```suggestion
            clear_array_input: 'Lijst wissen',
```

File: src/index.ts [hunk 2]
```diff
@@ -8,37 +8,38 @@ const dutchMessages: TranslationMessages = {
...
             show: 'Tonen',
             sort: 'Sorteren',
+            toggle_theme: 'Thema wisselen',
             undo: 'Ongedaan maken',
-            unselect: 'Deselecteren',
-            expand: 'Uitklappen',
-            close: 'Sluiten',
-            open_menu: 'Menu openen',
-            close_menu: 'Menu sluiten',
+            unselect: 'Unselect',
```
Comment (nickwaelkens) [inline src/index.ts:41]:
```suggestion
            unselect: 'Deselecteer',
```

**Review by nickwaelkens — COMMENTED** [review]
@kristelvdakker Thanks a lot for adding these! WDYT about these changes, before merging?

**Comment by kristelvdakker** [comment]
Yes! These are great, thank you 😄

**Comment by nickwaelkens** [comment]
@kristelvdakker Done and done! `v4.6.1.` is available now, thanks again 👍🏻

**Pull request merged**